tests/TestCase/Validation/ValidationTest.php

```diff
@@ -1646,6 +1646,12 @@ public function testTime()
         $this->assertTrue(Validation::time('1:00pm'));
         $this->assertFalse(Validation::time('13:00pm'));
         $this->assertFalse(Validation::time('9:00'));
+        $this->assertFalse(Validation::time('1'));
```

Comment:
Does 0 count as a valid time now too?

Reply:
If this is a back-port, please copy all of `testTime()`.

Reply:
@markstory @othercorey 
Yes, this is a back-port from https://github.com/cakephp/cakephp/pull/14969
I updated pull request description.